features/src/main/feature/camel-features.xml

```diff
@@ -92,10 +92,16 @@
 
     <feature name="jackson" version="${jackson2-version}">
         <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}$overwrite=merge</bundle>
```

Comment:
Why do you need to do that?

Reply:
I've just read your description, so I understand but we really need to add a comment because otherwise it will be removed at some point